### PR TITLE
Added new method install_parser, added parsing status files

### PR DESCRIPTION
**Pull request by lamarwhitej**
Added new method install_parser, added parsing status files.  Fixed issue with already installed elastic-benchmark by removing it if it is there then reinstalling.  It should do this anyway in case there have been updates.

### CODE REVIEW

File: jenkins/osa_onmetal.functions.groovy [hunk 1]
```diff
@@ -438,31 +438,42 @@ def aggregate_parse_failed_smoke(host_ip, results_file, elasticsearch_ip) {
...
 }
 
-
-def parse_results() {
+def install_elastic_parser() {
 	
     sh """
+    rm -rf elastic-benchmark
     git clone https://github.com/osic/elastic-benchmark
     sudo pip install -e elastic-benchmark
```
Comment (castulo) [inline jenkins/osa_onmetal.functions.groovy:464]: Why are you installing the package in development mode? This should be installed as an official package now:
pip install elastic-benchmark/

File: jenkins/osa_onmetal.functions.groovy [hunk 2]
```diff
@@ -438,31 +438,42 @@ def aggregate_parse_failed_smoke(host_ip, results_file, elasticsearch_ip) {
     '''
```
Comment (castulo) [inline jenkins/osa_onmetal.functions.groovy:438]: Why are you also installing elastic-benchmark here manually?

**Review by castulo — CHANGES_REQUESTED** [review]
I added a couple of comments that need to be addressed first.

**Comment by lamarwhitej** [comment]
Hey Cas let's just close this one.  I have some file changes that I need to get added.  I changed some file names in the during tests.  The pipeline is expecting different ones.

**Pull request closed without merging**